kornia/augmentation/container/image.py

```diff
@@ -35,29 +37,35 @@ class ImageSequential(nn.Sequential):
             If False, the whole list of args will be processed as a sequence in original order.
 
     Returns:
-        the tensor (, and the transformation matrix) has been sequentially modified by the args.
+        TensorWithTransMat: the tensor (, and the transformation matrix)
+            has been sequentially modified by the args.
```

Comment:
don't understand this sentence. The return of this container is the transformed image and the transform matrix, right ?

Reply:
The parenthesis meant optionally. LOL. I will try make it more clear.